src/Raven.Client/Documents/Session/Tokens/SuggestToken.cs

```diff
@@ -13,7 +13,7 @@ public class SuggestToken : QueryToken
         private SuggestToken(string fieldName, string alias, string termParameterName, string optionsParameterName)
         {
             FieldName = fieldName ?? throw new ArgumentNullException(nameof(fieldName));
-            _alias = alias;
+            _alias = alias?.Contains(" ") == true ? $"\"{alias}\"" : alias;
```

Comment:
insted of doing this, you need to use `QueryFieldUtil.EscapeIfNecessary` in the `Create` method. Check how this is done in FacetToken.

Reply:
done